packages/rancher-istio/charts/templates/istio-uninstall-job.yaml

```diff
@@ -29,6 +29,9 @@ spec:
             mountPath: /app/overlay-config.yaml
             subPath: overlay-config.yaml
           {{ end }}
+          securityContext:
+            runAsUser: 101
+            runAsGroup: 101 
```

Comment:
same comment as above re: choice of user / group

Reply:
see answer to other security context question.